support-models/src/test/scala/com/gu/support/promotions/SerialisationSpec.scala

```diff
@@ -111,9 +115,11 @@ class SerialisationSpec extends AsyncFlatSpec with SerialisationTestHelpers with
       appliesTo,
       "",
       Map[Channel, Set[PromoCode]]("" -> Set()),
-      DateTime.now(),
+      DateTime.now(ISOChronology.getInstanceUTC),
```

Comment:
DateTime.now() uses ISO chronology/London by default, but when you parse an ISO date you get the ISO chronology/UTC.  This means it fails the test despite seeming to be the same.

Reply:
Dates - always a problem!